service/src/main/java/org/apache/griffin/core/integration/GriffinEventListeners.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.griffin.core.integration;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+@Configuration
+@ConfigurationProperties(prefix = "internal.event")
+public class GriffinEventListeners {
+    private static final Logger LOGGER = LoggerFactory
+        .getLogger(GriffinEventListeners.class);
+    @Value("${internal.event.listeners}")
+    private String listeners;
```

Comment:
We might need to use List<String> as listeners.
In Spring, they will inject string array values as List<String>, as clz1,clz2,clz3

Reply:
let me try spring list